packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -906,15 +973,65 @@ export function getParameters(
 }
 
 /**
- * Returns an array of server objects for the opeartion at the given path and
+ * Returns an hash containing the callbacks defined in the given endpoint.
+ */
+export function getEndpointCallbacks(
+  path: string,
+  method: string,
+  oas: Oas3,
+  data: PreprocessingData
+): { [key: string]: CallbackObject } {
+  const callbacks = {}
+  const endpoint: OperationObject = oas.paths[path][method]
+
+  if (typeof endpoint.callbacks === 'object') {
+    let callbacksObject: CallbacksObject = endpoint.callbacks
+    for (let callbackName in callbacksObject) {
+      if (typeof callbacksObject[callbackName] === 'object') {
+        let callbackObject: CallbackObject | ReferenceObject =
+          callbacksObject[callbackName]
+        // Make sure we have CallbackObject:
+        if (typeof (callbackObject as ReferenceObject).$ref === 'string') {
+          callbackObject = resolveRef(
```

Comment:
Do we need to check for references here? If I recall correctly, the only time we check for references is dealing with schemas that may be contained in the components object ([related link](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#componentsObject)). 

Although, now that I've done some digging, I see that there are many other references we do not check for in general. Might need to file an issue about this.

Reply:
It allows to pseudo validate `CallbacksObject` and also to recompose them since they could be $ref to describe the `CallbackObject` and/or the `PathItemObject`.
Then we can safely go deeper to parse the operation that each `PathItemObject` contains in `processOperationCallbacks`.

Maybe i got it wrong ?

Reply:
Since `getEndpointCallbacks` is called before `processOperationCallbacks` and that the last one needs information about the whole callback content to compose the callback operation that is saved in `PreprocessingData`, i thought it was required to resolve all the potential references.
What would be the alternative then ?